codewiki/src/be/dependency_analyzer/dependency_graphs_builder.py

```
@@ -27,18 +47,14 @@ def build_dependency_graph(self) -> tuple[Dict[str, Any], List[str]]:
 
         # Prepare dependency graph path
         repo_name = os.path.basename(os.path.normpath(self.config.repo_path))
-        sanitized_repo_name = ''.join(c if c.isalnum() else '_' for c in repo_name)
+        sanitized_repo_name = "".join(c if c.isalnum() else "_" for c in repo_name)
         dependency_graph_path = os.path.join(
-            self.config.dependency_graph_dir, 
-            f"{sanitized_repo_name}_dependency_graph.json"
+            self.config.dependency_graph_dir, f"{sanitized_repo_name}_dependency_graph.json"
         )
         filtered_folders_path = os.path.join(
```

Ruff (0.14.10)

Local variable `filtered_folders_path` is assigned to but never used

Remove assignment to unused variable `filtered_folders_path`

(F841)